composer.json

```diff
@@ -34,21 +34,23 @@
         }
     },
     "require": {
-        "php": "^7.3 || ~8.0.0 || ~8.1.0",
+        "php": "~7.4.0 || ~8.0.0 || ~8.1.0",
         "laminas/laminas-cache-storage-implementation": "1.0",
         "laminas/laminas-eventmanager": "^3.4",
         "laminas/laminas-servicemanager": "^3.7",
         "laminas/laminas-stdlib": "^3.6",
+        "lcobucci/clock": "^2.0",
         "psr/cache": "^1.0",
         "psr/simple-cache": "^1.0",
         "webmozart/assert": "^1.9"
     },
     "require-dev": {
+        "cache/integration-tests": "dev-master",
```

Comment:
Hmm, we use `dev-master` only on this package? :thinking:

Reply:
its being removed with #199 
Since this work was based on early #199 state, I will rebase after #199 is merged

Reply:
```suggestion
        "cache/integration-tests": "dev-master",
        "lcobucci/clock": "^2.0",
```